tutorials/W1D1_Instructions/W1D1_Tutorial3.ipynb

```diff
@@ -0,0 +1,515 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2a21efb9",
+   "metadata": {
+    "execution": {}
+   },
+   "source": [
+    "<a href=\"https://colab.research.google.com/github/neuromatch/course-content-template/blob/main/tutorials/W1D1_Template/W1D1_Tutorial3.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a> &nbsp; <a href=\"https://kaggle.com/kernels/welcome?src=https://raw.githubusercontent.com/neuromatch/course-content-template/main/tutorials/W1D1_Template/W1D1_Tutorial3.ipynb\" target=\"_parent\"><img src=\"https://kaggle.com/static/images/open-in-kaggle.svg\" alt=\"Open in Kaggle\"/></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e79e6195",
+   "metadata": {
+    "execution": {}
+   },
+   "source": [
+    "# Tutorial 3: Content Enhancements\n",
+    "\n",
+    "**Week 1, Day 1: Instructions**\n",
+    "\n",
+    "**By Neuromatch**\n",
+    "\n",
+    "__Content creators:__ Konstantine Tsafatinos\n",
+    "\n",
+    "__Content reviewers:__ Konstantine Tsafatinos\n",
+    "\n",
+    "__Production editors:__ Konstantine Tsafatinos\n",
+    "\n",
+    "<br>\n",
```

Comment:
you can change content creators to your name if you like. I may give an example of how multiple people are creating the course content